packages/story-editor/src/utils/usePageAsCanvas.js

```diff
@@ -95,9 +98,15 @@ function usePageAsCanvas() {
         if (!pageCanvasPromise && fullbleedContainer) {
           import(/* webpackChunkName: "chunk-html-to-image" */ 'html-to-image')
             .then((htmlToImage) => {
+              const filter = (node) => {
+                // exclude selected element from generated image to prevent interfereing with contrast calculation. true includes and false excludes
+                return node?.dataset?.elementId !== selectedElementIds[0];
+              };
+
               const promise = htmlToImage.toCanvas(fullbleedContainer, {
                 fontEmbedCss: '',
                 pixelRatio: 1,
+                filter: selectedElementIds[0] && filter,
```

Comment:
I feel like `selectedElementIds.length === 1` is a better test. Also please move to a const like `hasSingleSelection` or similar.